packages/bulkapi/src/controllers/BulkApi.ts

```diff
@@ -13,7 +13,7 @@ import {
     lionwebResponse,
     ResponseMessage,
     DeletePartitionsResponse,
-    StoreResponse, HttpClientErrors, HttpSuccessCodes, getStringParam, getIntegerParam, isParameterError
+    StoreResponse, HttpClientErrors, HttpSuccessCodes, getStringParam, getIntegerParam, isParameterError, nextRepoVersion, getRepoVersion
```

Comment:
I'd think that `prettier` would re-format this import list.

Reply:
We have an issue for this: [https://github.com/LionWeb-io/lionweb-repository/issues/28](https://github.com/LionWeb-io/lionweb-repository/issues/28)

Reply:
Fixed